crates/control/src/motion/joint_command_sender.rs

```diff
@@ -90,6 +91,9 @@ impl JointCommandSender {
 
         context.positions.fill_if_subscribed(|| positions);
         context.stiffnesses.fill_if_subscribed(|| stiffnesses);
+        context
+            .positions_difference
+            .fill_if_subscribed(|| current_positions - positions);
```

Comment:
More intuitive would be a SHOULD - IS, not IS - SHOULD.

Reply:
Thanks, fixed.